--- miniz-3.0.2/miniz.pc.in.orig	2023-01-15 12:55:58.000000000 +0100
+++ miniz-3.0.2/miniz.pc.in	2024-01-30 22:01:14.605053896 +0100
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@/@PROJECT_NAME@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@/@PROJECT_NAME@
 
 Name: @PROJECT_NAME@
 Description: @PROJECT_DESCRIPTION@
